implementations/rust/ockam/ockam_transport/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ockam_transport"
```

Comment:
Sorry, I think I mislead you. We decided that ockam_transport_core would be better name, could you please rename it?

Reply:
No problem! Done :+1: